### DISCUSSION TITLE
getAllParametersMatching function in ospsuite R package

### DISCUSSION THREAD

**Discussion by moPellowe** [Open-Systems-Pharmacology/Forum#1515] [Questions & Problems]
I am trying to get all of the parameters "P (plasma<-> interstitial)" from the pkml file, but the function is not working as I expected. I know that there are parameters matching this name in the simulation, but they are not being found by the function. Is there something wrong with my function call?
![image](https://user-images.githubusercontent.com/57540473/234883167-4d9eca8f-846d-4369-a109-080b5a07c56b.png)

**Comment by Yuri05** [comment]
Cannot reproduce:. E.g. with the attached model: [simulation.zip](https://github.com/Open-Systems-Pharmacology/Forum/files/11345623/simulation.zip). Which package version (ospsuite) are you using?

```
> sim=loadSimulation("sim.pkml")
> getAllParametersMatching("Neighborhoods|Brain_pls_Brain_int|Alfentanil|P (plasma<->interstitial)", sim)
[[1]]
Parameter: 
   Path: Neighborhoods|Brain_pls_Brain_int|Alfentanil|P (plasma<->interstitial) 
   Value: 0.00 [dm/min] 
   isFormula: TRUE 
   formula: P/2 
   Value overrides formula: FALSE 
   isStateVariable: FALSE 

> getAllParametersMatching("Neighborhoods|**|P (plasma<->interstitial)", sim)
[[1]]
Parameter: 
   Path: Neighborhoods|Bone_pls_Bone_int|Alfentanil|P (plasma<->interstitial) 
   Value: 10.00 [dm/min] 
   isConstant: TRUE 
   isStateVariable: FALSE 

[[2]]
Parameter: 
   Path: Neighborhoods|Brain_pls_Brain_int|Alfentanil|P (plasma<->interstitial) 
   Value: 0.00 [dm/min] 
   isFormula: TRUE 
   formula: P/2 
   Value overrides formula: FALSE 
   isStateVariable: FALSE 

[[3]]
Parameter: 
   Path: Neighborhoods|Fat_pls_Fat_int|Alfentanil|P (plasma<->interstitial) 
   Value: 10.00 [dm/min] 
   isConstant: TRUE 
   isStateVariable: FALSE 

[[4]]
```

**Reply by moPellowe** [reply]
It looks like I am using ospsuite v9.1.4.

**Comment by msevestre** [comment]
I'd strongly suggest updating to newer version. The code you are running is over 2years old and we have fixed a lot of issues
